### DISCUSSION TITLE
Thread Parameter Adjustment

### DISCUSSION THREAD

**Discussion by Wangzhuozhu** [vdemichev/DiaNN#322] [Q&A]
We use DIA-NN on 96-thread win-server but the thread parameter cannot be adjusted higher than 48. During running we observed there were only 48 threads working. Is there any method to promote the DIA-NN performance on win-server?

**Comment by vdemichev** [comment]
Not sure why it's happening, maybe some platform limitation. I would just suggest to run two copies of DIA-NN in parallel on the same machine - this will bemore efficient anyway.

Vadim

**Reply by Wangzhuozhu** [reply]
Thanks! Let me try.
